tests/benchmarks/multimodal/jsonl/test_main.py

```
@@ -0,0 +1,106 @@
+                "--window-size", "5",
+                "--image-size", "32", "32",
+                "--image-dir", str(tmp_path / "imgs"),
+                "--seed", "42",
+                "-o", str(tmp_path / "sw.jsonl"),
+            ],
+        )
+
+        assert len(lines) == 6  # 2 users x 3 turns
+
+        # Round-robin interleaving: user_0, user_1, user_0, user_1, ...
+        session_ids = [l["session_id"] for l in lines]
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)